Translate Home.md into traditional Chinese

Victerose opened the pull request:
Translate Home.md into traditional Chinese
translator: Wang Hsiang (Taiwanese MSP 12nd)
translate.date: 09/30/2017

azurecla commented:
Hi __@MSFTMan__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (deonhe). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, or work for Microsoft Open Technologies, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

Victerose committed:
Translate Home.md into traditional Chinese

Translate Home.md into traditional Chinese
translator: Wang Hsiang (Taiwanese MSP 12nd)
translate.date: 09/30/2017
